include/CommandHandler/src/CommandHandler.cpp

```diff
@@ -288,8 +294,7 @@ void CommandHandler::HandleVrfy(SocketWrapper& socket_wrapper, const std::string
                     response += user.user_name + "\r\n";
```

Comment:
user_info_list is vector of user while we are expecting only one user. Moreover, ```RetrieveUserInfo()``` returns User object that has unnecessary information about hash_password and host_name. I think, you should ask team resposible for database rewrite ```RetrieveUserInfo()```  or create new method, and then change here the logic according new method.